dataops/gcp/gcp_setup_template.yml

```diff
@@ -1,12 +1,28 @@
 variables:
+  GCP_KEY_PATH: TBS
+  GCP_DOCKER_KEY_PATH: TBS
+  GIT_KEY_PATH: TBD
   DOCKER_REGISTRY: europe-central2-docker.pkg.dev
   BLOB_CONFIG_PATH: blob_args.json
 
-.setup:
+.gcp_setup: &gcp_setup
+  - gcloud --quiet components update
+  - gcloud auth activate-service-account --key-file ${GCP_KEY_PATH}
+  - echo "{\"token\":\"GCP_KEY_PATH\"}" > $BLOB_CONFIG_PATH
+
+.docker_setup: &docker_setup
+  - gcloud auth configure-docker $DOCKER_REGISTRY
+
+.git_setup: &git_setup
+  - chmod 600 $GIT_KEY_PATH
+
+.dev_setup:
```

Comment:
Why `.dev_setup` is without `*git_setup`? It's because `dev_release` don't contains `dp publish --key-path $GIT_KEY_PATH`?

Reply:
yes, git push is not used there